swarmzero_config_example.toml

```diff
@@ -1,3 +1,5 @@
+max_iterations = 10
+
```

Comment:
also update the test config - https://github.com/swarmzero/swarmzero/blob/main/tests/swarmzero_config_test.toml